src/SerialLoops/Dialogs/BgmLoopPropertiesDialog.cs

```diff
@@ -0,0 +1,233 @@
+                Value = LoopPreview.GetTimestampFromSample(LoopPreview.StartSample),
+            };
+            NumericStepper endSampleBox = new()
+            {
+                MinValue = startSampleBox.Value,
+                MaxValue = LoopPreview.GetTimestampFromSample((uint)(LoopPreview.Wave.Length / LoopPreview.Wave.WaveFormat.BitsPerSample * 8 / LoopPreview.Wave.WaveFormat.Channels)),
+                MaximumDecimalPlaces = 4,
+                Value = LoopPreview.GetTimestampFromSample(LoopPreview.EndSample),
+            };
+            startSampleBox.MaxValue = endSampleBox.Value;
+
+            startSampleSlider.ValueChanged += (obj, args) =>
+            {
+                if (startSampleSlider.Value > endSampleSlider.Value)
+                {
+                    startSampleSlider.Value = endSampleSlider.Value;
+                    return;
+                }
+                LoopPreview.StartSample = (uint)startSampleSlider.Value;
+                startSampleBox.Value = LoopPreview.GetTimestampFromSample(LoopPreview.StartSample);
+                endSampleBox.MinValue = startSampleBox.Value;
+                LoopPreviewPlayer.Stop();
+            };
+            endSampleSlider.ValueChanged += (obj, args) =>
+            {
+                if (endSampleSlider.Value < startSampleSlider.Value)
+                {
+                    endSampleSlider.Value = startSampleSlider.Value;
+                    return;
+                }
+                LoopPreview.EndSample = (uint)endSampleSlider.Value;
+                endSampleBox.Value = LoopPreview.GetTimestampFromSample(LoopPreview.EndSample);
+                startSampleBox.MaxValue = endSampleBox.Value;
+                LoopPreviewPlayer.Stop();
+            };
+            startSampleBox.ValueChanged += (obj, args) =>
+            {
+                LoopPreview.StartSample = LoopPreview.GetSampleFromTimestamp(startSampleBox.Value);
+                startSampleSlider.Value = (int)LoopPreview.StartSample;
+                endSampleBox.MinValue = startSampleBox.Value;
+                LoopPreviewPlayer.Stop();
+            };
+            endSampleBox.ValueChanged += (obj, args) =>
+            {
+                LoopPreview.EndSample = LoopPreview.GetSampleFromTimestamp(endSampleBox.Value);
+                endSampleSlider.Value = (int)LoopPreview.EndSample;
+                startSampleBox.MaxValue = endSampleBox.Value;
+                LoopPreviewPlayer.Stop();
+            };
+
+            Button saveButton = new() { Text = "Save" };
+            Button cancelButton = new() { Text = "Cancel" };
+            saveButton.Click += (obj, args) =>
+            {
+                SaveChanges = true;
+                Close();
+            };
+            cancelButton.Click += (obj, args) =>
+            {
+                Close();
+            };
+
+            Content = new StackLayout
+            {
+                Orientation = Orientation.Horizontal,
+                Spacing = 5,
```

Comment:
```suggestion
                Spacing = 5,
                Padding = 10,
```
This needs just a smidge of padding so the Pause/Stop buttons aren't giving the window controls a hug of death